README.md

```diff
@@ -9,48 +9,61 @@ We are trying to create a medium where people who want to start with contributin
+1. The `master` branch is automatically deployed [to opensourcehelpcommunity.herokuapp.com](http://opensourcehelpcommunity.herokuapp.com/) .
 
 This is using [Django(1.11)](https://www.djangoproject.com/) and [Bootstrap](http://getbootstrap.com/)
 
 ## How to Contribute
-This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests submitting pull requests as well. This project adheres to the Collaborative [code of conduct](https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/blob/develop/CODE_OF_CONDUCT.md). By participating, you expect to maintain the code of conduct. Before creating `New issue` and `Pull request`, please refer to the [template](docs).  
+
+This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests submitting pull requests as well. This project adheres to the Collaborative [code of conduct](https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/blob/develop/CODE_OF_CONDUCT.md). By participating, you expect to maintain the code of conduct. Before creating `New issue` and `Pull request`, please refer to the [template](docs).
 
 All the development is done on `predev` branch and once we're ready for testing the deployment in real environment with databases we merge the `predev` branch with the `develop` branch. When we're ready for new release we merge `develop` with `master` to deploy it on our main website. **Please submit your pull request based on `predev` branch.**
 
 ## Installations
+
 Run
-```
+
+```bash
 pip install -r requirements.txt
 ```
-to install everything required to run this project on heroku as well as on your local.
 
+to install everything required to run this project on heroku as well as on your local.
 
 ## To run this in your local
 
 1. Clone this repository using
-	```
-	git clone git@github.com:OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.git
-	```
-
-2. Go inside main Django app [Instructional video on installing Django](https://youtu.be/qgGIqRFvFFk)
-	```
-	cd oshc
-	```
-
-3. Collectstatic files using
-	```
-	python manage.py collectstatic
-	```
-
-4. Run the app
-	```
-	python manage.py runserver
-	```
+
+   ```bash
+   git clone git@github.com:OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.git
+   ```
+
+1. Go inside main Django app [Instructional video on installing Django](https://youtu.be/qgGIqRFvFFk)
+
+   ```bash
+   cd oshc
+   ```
+
+1. Install [pre-commit](pre-commit.com) into your git hooks. [pre-commit](pre-commit.com) will now run on every commit. Every time you clone a project that is using [pre-commit](pre-commit.com) running [pre-commit](pre-commit.com) install should always be the first thing you do after installing requirements.
+
+   ```bash
+   pre-commit install
+   ```
+
+1. Collectstatic files using
+
+   ```bash
+   python manage.py collectstatic
+   ```
+
+1. Run the app
+   ```bash
```

Comment:
thanks for this cleanup! 🎉  I was planning to do this from quite a while but didn't.